### ISSUE TITLE
aws sdk mediainfo with space

### ISSUE THREAD

**Issue by mrnobody1988** [MediaArea/MediaInfoLib#392]
Hi,

Please find below.
http://stackoverflow.com/questions/42255222/how-to-use-mediainfo-on-aws-s3-filname-contain-space.

Thanks

**Comment by JeromeMartinez** [comment]
Copy/paste in case link disappears.

***

How to use mediainfo on aws s3 filename contain space?

I got this error. E: https://s3-eu-west-1.amazonaws.com/test/video/batmanvs superman.mov, HttpVersionNotSupportedThe HTTP version specified is not supported.

I tried single quote and double quote. mediainfo 'https://s3-eu-west-1.amazonaws.com/test/video/batmanvs superman.mov'

I got the same error.

But if the filename without spacing it works fine.
